package.json

```diff
@@ -77,6 +77,7 @@
     "react-addons-css-transition-group": "0.14.3",
     "react-addons-linked-state-mixin": "0.14.3",
     "react-addons-update": "0.14.3",
+    "react-click-outside": "2.1.0",
```

Comment:
Missing license in CREDITS.md:

https://github.com/kentor/react-click-outside/blob/master/LICENSE.txt